charts/all/openvino/templates/ov-notebookinstance.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: intel.com/v1alpha1
+kind: Notebook
+metadata:
```

Comment:
Would it make sense to make some of these chart parameters, to allow for re-use of the chart?

Reply:
Yes, that makes sense. I did some changes here and for rhods-datasciencecluster